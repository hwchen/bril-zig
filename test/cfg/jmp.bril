../../bril/test/interp/core/jmp.bril