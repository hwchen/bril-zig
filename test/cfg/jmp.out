{"b000":["somewhere"],"b001":["somewhere"],"somewhere":[]}
